src/osd/PG.cc

```diff
@@ -1944,6 +1944,32 @@ void PG::queue_op(OpRequestRef& op)
   }
 }
 
+void PG::queue_op_without_pg_lock(OpRequestRef& op)
```

Comment:
As I've said, you need *some* lock in order to maintain reply ordering.  Once you identify and name that lock, update these names.